hugoserver.sh

```diff
@@ -21,7 +21,7 @@ else
 fi
 
 # You can set HUGO_IMAGE in the env to overwrite or it will default to this version
-HUGO_IMAGE=${HUGO_IMAGE:-cibuilds/hugo:0.121.2}
+HUGO_IMAGE=${HUGO_IMAGE:-cibuilds/hugo:0.123.2} # note! this is one patch behind prod bc the tag isn't there yet
```

Comment:
It seems to exist now: https://hub.docker.com/layers/cibuilds/hugo/0.123.3/images/sha256-687713ea5b62ee5b34e6cd93155aa05df06617c41e00c945c0e8c2af767084f0?context=explore

```suggestion
HUGO_IMAGE=${HUGO_IMAGE:-cibuilds/hugo:0.123.3}
```